style.css

```diff
@@ -0,0 +1,103 @@
+/* general page style  */
+*{
+    font-family: sans-serif;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.services_section{
+    text-align: center;
+    background-color: #cee6ea;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+/* content style */
+.main_div{
+   display: flex;
+   justify-content: center;
+   align-items: center;
+   /* border: 1px solid; */
+   width: 100%;
+   height: 33rem;
+}
+.img{
+    height: 170px;
+    object-fit: cover;
+    background-color: #cee6ea;
+    border-radius: 50%;
+    
+}
+.title{
+    margin: 20px;
+    color: #071144;
+    padding-top: 15px;
+    font-weight: bold;
+
+    
+}
+.descriptive_paragraph{
+    color: #071144;
+    font-size: 15px;
+    
+    width: 100%;
+
+    text-align: center;
+    
+    
+}
+.divs{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    display: inline-block;
+    text-align: center;
+}
+.button{
+    width: 180px;
+    height: 50px;
+    border-radius: 15%;
+    background-color: #0f4b5e;
+    color: white;
+    font-size: 20px;
+    font-weight: lighter;
+    margin-top: 10px;
+
+}
+.button:hover{
+    background-color: white;
+    color: cornflowerblue;
+}
+a:link{
+    text-decoration: none;
+}
+h2{
+    color: #071144;
+    text-align: center;
+}
+/* media quiery */
+@media screen and (max-width: 768px) {
+    .main_div{
+        display: flex;
+        flex-direction: column;
+        height: auto;
+
+    }
```

Comment:
**`display:flex` has already taken so, just you have to add the property `flex-direction`** 
```suggestion
    .main_div{
        flex-direction: column;
        height: auto;

    }
```